Update user profile docs for updatable root attributes

rkhoriander opened the pull request:
Update user profile docs to reflect new updatable attributes on user profile and in signup process.

rkhoriander committed:
Update manage-users-using-the-dashboard.md

rkhoriander committed:
Create change-user-attribute-update.md

rkhoriander committed:
Create new doc for updating user attributes

rkhoriander committed:
Update normalized-user-profile-schema.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update user-data-storage-scenario.md

rkhoriander committed:
Update normalized-user-profile-schema.md

rkhoriander committed:
Update update-user-profiles-using-your-database.md

rkhoriander committed:
Update read-metadata.md

rkhoriander committed:
Update normalized-user-profile-schema.md

rkhoriander committed:
Update normalized-user-profile-schema.md

rkhoriander committed:
Update read-metadata.md

rkhoriander committed:
Update manage-users-using-the-management-api.md

rkhoriander committed:
Update manage-users-using-the-management-api.md

rkhoriander committed:
Update manage-users-using-the-dashboard.md

rkhoriander committed:
Update change-user-pictures.md

rkhoriander committed:
Create _users_update_normalized_profile_attributes.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update manage-users-using-the-management-api.md

rkhoriander committed:
Update manage-users-using-the-dashboard.md

rkhoriander committed:
Update change-user-attribute-update.md

rkhoriander commented:
https://docs-content-staging-pr-7223.herokuapp.com/docs/users

rkhoriander commented:
@randynasson - Whenever you can provide the screenshot of the new Connection setting in the Dashboard, I can update the guide. Otherwise, I believe this is done. Let me know if you see anything you would like to change.

rkhoriander committed:
Adding new screenshot for dashboard

rkhoriander committed:
Update change-user-attribute-update.md

rkhoriander committed:
Update change-user-attribute-update.md

rkhoriander committed:
Update change-user-attribute-update.md

rkhoriander committed:
Update change-user-attribute-update.md

rkhoriander committed:
Added border

rkhoriander committed:
Update and rename change-user-attribute-update.md to allow-update-normalized-user-profile.md

rkhoriander committed:
Update _users_update_normalized_profile_attributes.md

rkhoriander committed:
Update change-user-pictures.md

rkhoriander committed:
Update read-metadata.md

rkhoriander committed:
Update normalized-user-profile-schema.md

rkhoriander committed:
Update normalized-user-profile-schema.md

rkhoriander committed:
Merge branch 'master' into rkhoriander-user-profile-updatable-attributes

rkhoriander committed:
Added signup changes

rkhoriander commented:
https://docs-content-staging-pr-7223.herokuapp.com/docs/users

rkhoriander committed:
Add caveat about Auth0 connections and update attributes on user guide

panga commented on articles/users/normalized/auth0/normalized-user-profile-schema.md:
```diff
@@ -22,7 +22,7 @@ Fields that are always generated:
 * **`user_id`**: the user's unique identifier. This is unique per Connection, but the same for all apps that authenticate via that Connection.
 
-A user's `name`, `nickname`, and `picture` attributes are not directly editable, however you can update the fields in the `user_metadata` to update them for your front-end as desired. [Click here for more details for changing a user's picture](/users/guides/change-user-pictures); the `name`, `nickname` fields can be updated in the same way.
+By default, a user's `name`, `nickname`, and `picture` attributes provided by identity providers other than Auth0 (such as Google, Facebook, Twitter) are not directly editable since they are updated from the identity provider each time a user logs in. If you want to be able to edit these attributes, you must [enable a connection to allow updates to the normalized user profile](/connections/guides ...
```
Please change line 22.

From:
`* **`nickname`**: the user's username.`

To:
`* **`nickname`**: the first part of the user's email or the user's username if using Auth0 store and require username.`

rkhoriander replied:
Done. Thanks!

randynasson reviewed (dismissed):
Over-all, these changes look good. We don't want to publish them until we are ready to release the feature though. Can we sit on these until we are ready to go live?

rkhoriander committed:
Requested change to nickname definition

rkhoriander commented:
@randynasson: Yes, I have a task in the current sprint, which runs until 3/8. And if you need to wait longer than that, we can move it to the next sprint. I'll continue to check in with you on Slack.

panga commented on articles/users/normalized/auth0/normalized-user-profile-schema.md:
```diff
@@ -18,11 +18,11 @@ The attributes that Auth0 maps to a common schema are listed below.
 
 * **`name`**: the user's full name.
-* **`nickname`**: the user's username.
+* **`nickname`**: the first part of the user's email, or the user's username if using the Auth0 data store and require username.
```
```suggestion
* **`nickname`**: the user's username or the local part of the user's email.
```

rkhoriander replied:
@panga: Are there conditions surrounding when each is used?

panga replied:
No, the nickname default value is:
We first try to use username, but if username is not present, we set the local part of the email.
That's why I used the `OR` there.

rkhoriander replied:
Updated. Thank you.

panga reviewed (changes requested):
I need to request changes again.

rkhoriander committed:
Updated with requested change to nickname field

Commit:
Custom signup fixes

tamigoodall committed:
Create device-app-auth-flow.md

Created doc 1 of 3

tamigoodall committed:
Added draft flow diagrams and how tos

Added draft flow diagrams and how to doc

tamigoodall committed:
Added more content

Rearranged diagrams
Added notes from source docs
Added links to Logs (TBD)
Updated index.md file under flows

Commit:
Naming and screen changes

Commit:
Add info about API

Commit:
Change option name

rkhoriander committed:
Create device microsite, rework flow doc, new image for microsite

rkhoriander committed:
merge master

rkhoriander committed:
Device Auth Initial Docs for review

rkhoriander committed:
Fix typo

rkhoriander committed:
Merge branch 'master' into device-apps-docs

rkhoriander committed:
Add new root attribute guides

rkhoriander committed:
Add guides and article linking to guides

rkhoriander committed:
Merge branch 'master' into rk-updatable-attribute-updates

rkhoriander committed:
Merge branch 'master' into rkhoriander-user-profile-updatable-attributes

rkhoriander committed:
Merge branch 'rk-updatable-attribute-updates' into rkhoriander-user-profile-updatable-attributes

rkhoriander committed:
Remove new device docs

rkhoriander committed:
Remove new device flow docs

rkhoriander committed:
Merge branch 'rk-updatable-attribute-updates' into rkhoriander-user-profile-updatable-attributes

rkhoriander committed:
Make changes to reflect changes in dashboard toggle functionality

rkhoriander committed:
Remove extraneous updates

rkhoriander committed:
Remove extraneous files

rkhoriander committed:
Fix app errors

rkhoriander committed:
Fix typo

rkhoriander committed:
Fix broken links; add keep reading section

rkhoriander commented:
https://docs-content-staging-pr-7223.herokuapp.com/docs/users/normalized/auth0/update-root-attributes

rkhoriander committed:
Bulk import/export updates

rkhoriander committed:
Merge branch 'master' into rkhoriander-user-profile-updatable-attributes

rkhoriander committed:
Add upsert info

rkhoriander commented:
Main pages changed for Bulk Import/Export:

- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/references/user-profile-structure
- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/guides/bulk-user-imports
- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/guides/bulk-user-exports
- https://docs-content-staging-pr-7223.herokuapp.com/docs/extensions/user-import-export
- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/references/bulk-import-database-schema-examples

rkhoriander committed:
Bulk import/attribute changes from feedback

rkhoriander committed:
Merge branch 'master' into rkhoriander-user-profile-updatable-attributes

tamigoodall committed:
Changed H2s to sentence case

Changed H2s to sentence case
